modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -265,34 +367,25 @@ Returns five records starting from the second position.
 
 === Server-Side Pagination in APIkit for OData 4
 
-Your API implementation retrieves a limited number of records using a fix `pageSize` parameter. The response of the first request contains `@odata.nextlink` with a URL to get the next page.
+Server-side pagination is a way of controlling a subset of data requests that were fetched from a client-side server using a fixed `pageSize` parameter. It divides the response data set into discrete pages to improve data readability. The server defines the page size, which is the maximum number of records returned by the server in a request.
 
-For example:
+===== How server-side-pagination works in OData 4
 
-[source,curl]
---
-curl -X GET ‘localhost:8081/api/Orders’
---
+OData defines a `@odata.nextLink` to indicate that a response is only a subset of the requested collection of entities. As such, `@odata.nextLink` contains a URL that allows the client to retrieve the next subset of the requested collection.
+Apart from that, the URL contains a query parameter `$skiptoken` that indicates the server where the next page starts.
 
-.Response
-[source,json]
---
-{
-	value: [{ set of records}],
-	“@odata.nextlink”: ‘localhost:8081/api/Orders?$skiptoken=10’
-}
---
+==== How to implement this feature in APIkit for OData 4
 
-In this example, `$skiptoken` is "10", because the `pageSize` parameter is configured to return ten records.
+<apikit-odata:request-entity-collection-listener> allows you to define a set of fields in the *Responses* tab, in Anypoint Studio, to implement the server-side pagination.
```

Comment:
- <apikit-odata:request-entity-collection-listener> allows > `<apikit-odata:request-entity-collection-listener>` enables... (add backticks ` to XML code )

Reply:
Requesting assistance from APIkit team to learn which are the XML elements.

Reply:
`apikit-odata:request-entity-listener`, `apikit-odata:request-entity-collection-listener`, `apikit-odata:serialize-entity-collection`, `apikit-odata:serialize-entity`, `apikit-odata:transform-to-sql-select`, `apikit-odata:route`. And `apikit-odata:config`

Reply:
Here is a full list:

```console
# Command to find all instances of apikit-odata:SOMETHING 
> git grep -oh apikit-odata:[^>/ ]\* | sort | uniq
apikit-odata:collection-success-response
apikit-odata:config
apikit-odata:entity-collection-success-response
apikit-odata:entity-success-response
apikit-odata:flow-response-metadata-properties
apikit-odata:flow-response-metadata-property
apikit-odata:http-request-parameters
apikit-odata:inbound-content
apikit-odata:raw-entity-collection-success-response
apikit-odata:raw-entity-success-response
apikit-odata:request-entity-collection-listener
apikit-odata:request-entity-listener
apikit-odata:request-parameters
apikit-odata:route
apikit-odata:serialize-entity
apikit-odata:serialize-entity-collection
apikit-odata:success-response
apikit-odata:transform-to-sql-select
```

Reply:
Change made! Thanks!